CODE/usb_history.py

```
@@ -0,0 +1,89 @@
+            if ctypes.windll.advapi32.RegOpenKeyExW(root_key, sub_key_path, 0, winreg.KEY_READ,
+                                                    ctypes.byref(handle)) != 0:
+                return None
+
+            ft = ctypes.wintypes.FILETIME()
+            if ctypes.windll.advapi32.RegQueryInfoKeyW(handle, None, None, None, None, None, None, None, None, None,
+                                                       None, ctypes.byref(ft)) != 0:
+                return None
+
+            t = ((ft.dwHighDateTime << 32) + ft.dwLowDateTime) // 10
+            return datetime(1601, 1, 1) + timedelta(microseconds=t)
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)